dhcpv4/ztpv4/parse_circuitid.go

```diff
@@ -37,6 +37,8 @@ var circuitRegexs = []*regexp.Regexp{
 	regexp.MustCompile("^Ethernet(?P<slot>[0-9]+)/(?P<port>[0-9]+)$"),
 	// Juniper bundle interface ae52.0
 	regexp.MustCompile("^ae(?P<port>[0-9]+).(?P<subport>[0-9])$"),
+	// Ciena interface format
+	regexp.MustCompile(`\.OSC(-[0-9]+)?-(?P<slot>[0-9]+)-(?P<port>[0-9]+)$`),
```

Comment:
Why ` instead of "? Can we keep the same quoting as above?

Reply:
I changed them because the linter was complaining about using the double escape

Reply:
https://deepsource.io/gh/faginawu/hugo/issue/SCC-S1007/occurrences